tasks/release.py

```diff
@@ -972,6 +972,72 @@ def finish(ctx, major_versions="6,7"):
     # Update internal module dependencies
     update_modules(ctx, str(new_version))
 
+    # Step 3: branch out, commit change, push branch
+
+    final_branch = f"{new_version}-final"
+
+    print(color_message(f"Branching out to {final_branch}", "bold"))
+    ctx.run(f"git checkout -b {final_branch}")
+
+    print(color_message("Committing release.json and Go modules updates", "bold"))
+    print(
+        color_message(
+            "If commit signing is enabled, you will have to make sure the commit gets properly signed.", "bold"
+        )
+    )
+    ctx.run("git add release.json")
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Could you please first check that the `release.json` files exist with smth like `os.path.isfile(path)` and return an error if it doesn't ?

It's an edge case but it'd be better to have it caught if it does happen.

Reply:
Guess it's fine yes !